device_doctor/lib/src/ios_device.dart

```diff
@@ -14,6 +14,11 @@ import 'health.dart';
 import 'mac.dart';
 import 'utils.dart';
 
+/// Identifiers for devices that should never be rebooted.
+final Set<String> noRebootForbidList = <String>{
```

Comment:
This name sounds like a double negative, maybe noRebootList? or rebootForbidList?

Reply:
Renamed to `noRebootList`.